extensions/podman/src/extension.ts

```diff
@@ -150,6 +164,17 @@ export async function updateMachines(provider: extensionApi.Provider): Promise<v
   if (installedPodman) {
     shouldCleanMachine = shouldNotifyQemuMachinesWithV5(installedPodman);
   }
+  // check if the machine needs to be cleaned for v4 --> v5 format
+  if (!shouldCleanMachine) {
+    shouldCleanMachine = isIncompatibleMachineOutput(machineListOutput.stderr);
```

Comment:
On Windows we never reach this point bc the `getJSONMachineList` function above fails and it throws

Reply:
fixed